### ISSUE TITLE
CircleCI build matrix

### ISSUE THREAD

**Issue by Korijn** [conda-forge/vtk-feedstock#6]
@jakirkham I've noticed that you added a https://github.com/dennybaa/circleci-helpers package to conda-forge. How do I get started using it here? Or does conda-smithy require work to make it accessible?

**Comment by jakirkham** [comment]
This is going to be a `conda-smithy` thing. There is still quite a bit more work before we have `conda-smithy` doing something with it. Given I have some other deadlines I need to meet by the end of the month. I probably won't have time to look at it again until next month.

**Comment by Korijn** [comment]
@jakirkham any developments?

For anyone reading this; Circle CI builds don't run concurrently yet, so we can't build all the VTK configurations within the allowed build time.

Supporting a build matrix on Circle CI would be the key to resolving this issue.

**Comment by jakirkham** [comment]
Would it be alright if we close this and move discussion to issue ( https://github.com/conda-forge/conda-smithy/issues/203 )? Sorry just want to make sure it doesn't get too hard to follow and keep up with.

**Comment by Korijn** [comment]
Done!

**Comment by jakirkham** [comment]
This can now be done with `conda-smithy` 2.4.0.

**Comment by Korijn** [comment]
Woohoo!

**Comment by jakirkham** [comment]
Please see PR ( https://github.com/conda-forge/vtk-feedstock/pull/39 ).
